### PR TITLE
ci: Add basic dep bot

### PR DESCRIPTION
**Pull request by jcs090218**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```
Comment (psibi) [inline .github/dependabot.yml:6]: Do you think we can have it monthly ? I'm scared of new PR's coming daily!

Comment (jcs090218) [reply .github/dependabot.yml]: Yeah, you are right. `monthly` is a better option! :)

**Review by psibi — APPROVED** [review]
LGTM, but I believe the CI is failing because of unrelated issues ?

**Comment by jcs090218** [comment]
> LGTM, but I believe the CI is failing because of unrelated issues ?

Yeah, the ELPA is down now. 🤔

**Pull request merged**